### DISCUSSION TITLE
hey, bro, what's your score?

### DISCUSSION THREAD

**Discussion by Looong01** [T-Atlas/EBU6304_G058_flight#144] [General]
hey, bro, what's your score?
[Group project final submission](https://qmplus.qmul.ac.uk/mod/assign/view.php?id=1922281)
![image](https://user-images.githubusercontent.com/69568351/173549114-bd784030-21d1-4a35-a426-0ee8c070e88b.png)

**Comment by T-Atlas** [comment]
Hi Looong01, 
Thank you for your interest in this project. You can fork our project repository for deeper research or contribute, and if you appreciate our team's work you can give us a star. As a potential future program developer, I think the knowledge, project development skills and most critically the team spirit learned in this project are more valuable than the final score! Just relax and have a common mind to appreciate each other and learn what is good from others' projects will help us to go further.
Good luck!

**Reply by Looong01** [reply]
Is that a automatic reply?

**Reply by T-Atlas** [reply]
Yes, you are right. I could be an AI program for auto-responding to issues.
